Import key value from GND Console (#198)
Macrometacorp/docs: docs/collections/_stream-tab.md

> Each collection in the GDN can be a stream. Collection streams use the WebSocket protocol to emit event messages for operations performed on the collection.

Reviewer: ```suggestion
Each collection in Macrometa GDN can be a stream. Collection streams use the WebSocket protocol to emit event messages for operations performed on the collection.
```

While it's technically correct, saying "the GDN" feels a bit weird to me.  I'm open to using it, but my instinct is to call it "Macrometa GDN" instead.

The Stream tab is only displayed if streams are enabled on that collection.

For more information about streams, refer to [Streams](../streams/index.md) and [Stream Workers](../cep/index.md).

## View a collection stream

1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
1. Click **COLLECTIONS**.

> 1. Click the collection that you want to view the stream of.

Reviewer: ```suggestion
1. Click the collection for which you want to view the stream.
```
